bower.json

```diff
@@ -16,7 +16,7 @@
     "tests"
   ],
   "dependencies": {
-    "angular-material": "0.10.0",
+    "angular-material": "0.9.0",
```

Comment:
@nick22891 I'm not sure why we downgraded the angular-material version here? Kindly advise.